src/handlers/tests/interventions-processor-lambda.test.ts

```diff
@@ -1,28 +1,89 @@
-import { handler } from '../interventions-processor-handler';
-import { ContextExamples } from '@aws-lambda-powertools/commons';
+import {handler} from '../interventions-processor-handler';
+import {ContextExamples} from '@aws-lambda-powertools/commons';
 import logger from '../../commons/logger';
-import type { SQSEvent, SQSRecord } from 'aws-lambda';
-import { logAndPublishMetric } from '../../commons/metrics';
-import { DynamoDatabaseService } from '../../services/dynamo-database-service';
-import { validateEvent, validateInterventionEvent } from '../../services/validate-event';
-import {
-  AccountStateEngine,
-} from '../../services/account-states/account-state-engine';
-import { getCurrentTimestamp } from '../../commons/get-current-timestamp';
-import { StateTransitionError, TooManyRecordsError, ValidationError } from '../../data-types/errors';
-import {MetricNames} from "../../data-types/constants";
+import type {SQSEvent, SQSRecord} from 'aws-lambda';
+import {logAndPublishMetric} from '../../commons/metrics';
+import {DynamoDatabaseService} from '../../services/dynamo-database-service';
+import {validateEvent, validateInterventionEvent} from '../../services/validate-event';
+import {AccountStateEngine,} from '../../services/account-states/account-state-engine';
+import {getCurrentTimestamp} from '../../commons/get-current-timestamp';
+import {StateTransitionError, TooManyRecordsError, ValidationError} from '../../data-types/errors';
+import {EventsEnum, MetricNames, TICF_ACCOUNT_INTERVENTION} from "../../data-types/constants";
+import {sendAuditEvent} from "../../services/send-audit-events";
```

Comment:
for some reason the imports got a bit messed up, missing the whitespaces

Reply:
ahah yeah - I am of the opinion that we should have our liniting covering our test files too, otherwise it is gonna become a faff to address those things everytime (no spaces around curly braces for my editor)